src/components/AddGame/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react"
+import { SearchBar } from "../SearchBar"
+import { ResultList } from "../ResultList"
+import { Item, options } from "../../utils/searchSubmit"
+import { FormikActions } from "formik"
+import { validateToken } from "../../utils/validateToken"
+import xml, { ElementCompact } from "xml-js"
+import Axios from "axios"
+
+interface State {
+  items: Item[]
+}
+
+export type SearchSubmit = (
+  values: { search: string },
+  actions: FormikActions<{ search: string }>,
+) => Promise<void | Item[] | undefined>
+
+const defaultState: State = {
+  items: [],
+}
+
+export class AddGame extends React.Component {
+  public state = defaultState
+
+  public onSearchSubmit = async (
+    values: {
+      search: string
+    },
+    actions: FormikActions<{
+      search: string
+    }>,
+  ) => {
+    const { setStatus, setSubmitting } = actions
+    const { search } = values
+    setStatus(null)
+    //Set up await syntax
+    if (validateToken(localStorage.jwtToken)) {
+      const result = await Axios.get(
+        `http://localhost:8080/https://www.boardgamegeek.com/xmlapi2/search?query=${search}&type=boardgame`,
```

Comment:
It would be good to globalize (or at least hoist) these root domains to make it more flexible for non-dev environments.